general-concepts/dependencies/text.xml

```diff
@@ -461,6 +461,12 @@ DEPEND="!build? (
 <title>Any of Many Dependencies</title>
 <body>
 
+<note>
+To ease dependency resolution for Portage, it is recommended that you sort
+the elements in <e>preferred</e> order first. See
+<uri link="https://bugs.gentoo.org/489458">this bug</uri> for more context.
```

Comment:
I'd prefer "[bug 489458](https://bugs.gentoo.org/489458)" instead of "[this bug](https://bugs.gentoo.org/489458)" here. I'll update this when merging, if you don't mind.

Reply:
Completely fine with me. I'll keep this style preference in mind in future.